builders/tools/precompiles/xc20.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Cross-Chain Assets
+description:  Learn how to access and interact with an ERC-20 interface for cross-chain tokens on Moonbeam, and check how this is done on the Moonbase Alpha TestNet by using the assets precompiled contract.
+---
+
+#  Cross-Chain Assets Precompile
+
+![Cross-Chain Assets Precompiled Contracts Banner](/images/builders/tools/precompiles/xc20/xc20-banner.png)
+
+## Introduction {: #introduction } 
+
+The [Cross-Consensus Message (XCM)](https://wiki.polkadot.network/docs/learn-crosschain) format defines how messages can be sent between blockchains. This format opens the door to transfer messages and assets between Moonbeam and the Relay Chain or other parachains.
+
+Cross-chain assets on Moonbeam are one-to-one representations of the origin chain’s native tokens and are known as XC-20s. For example, Kusama's KSM representation will be known as XC-KSM. XC-20s allow users to bring their tokens to Moonbeam and interact with them through a standard ERC-20 interface via the [assets precompile](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/ERC20.sol).
+
+Under the hood, this is made possible by Substate’s assets pallet, which is coded in Rust. However, no knowledge of Rust is necessary as the assets precompile enables Solidity developers to use Substrate functions to interact with cross-chain assets through an easy-to-use and familiar format. Please note that that the precompile does not support transfers. This is intentionally done to stay as close as possible to the standard ERC-20 interface. <TODO: link to other XCM page with more information>
```

Comment:
Third paragraph should be how the feature being introduced solves problems with the current standard.

"XC-20s allow users/developers to access these interoperable assets through a familiar ERC-20 interface etc"

Reply:
"Sell more the idea of the XC-20, it is a pretty elegant solution for developers"